sdk/search/search/README.md

```diff
@@ -400,6 +400,121 @@ for (const result of updateResult.results) {
 }
 ```
 
+### 3. Create and authenticate a `SearchServiceClient`
+
+You could use the following code to create and authenticate a `SearchServiceClient`. 
+
+```js
+import { SearchServiceClient, SearchApiKeyCredential }  from "@azure/search";
+
+const endPoint: string = "<endpoint>";
+const apiKey: string = "<apikey>";
+
+const credential: SearchApiKeyCredential = new SearchApiKeyCredential(apiKey);
```

Comment:
`SearchApiKeyCredential` doesn't exist in master anymore, since we moved to `AzureKeyCredential`

Reply:
I do not understand this one alone. Because if I use the code:

```
const client = new SearchServiceClient(
    "<endpoint>",
    new AzureKeyCredential("<apikey>")
  );
```

I get the error ```TypeError: AzureKeyCredential is not a constructor```

Reply:
have you merged master recently?

Reply:
That resolved it. Changed it